app/actions/ModActionPanel/QuickAction.tsx

```diff
@@ -560,9 +565,13 @@ function Form(
                     <LastReviewedTimestamp subjectStatus={subjectStatus} />
                   </p>
                   {!!subjectStatus.comment && (
-                    <Card hint="important" className="mt-2">
-                      <strong>Note:</strong> {subjectStatus.comment}
-                    </Card>
+                    <div className="mt-2">
```

Comment:
kinda unrelated but mods wanted to give comments more alarming look since we mostly use these comments for critical instruction to other mods.